src/Plugin.php

```diff
@@ -86,6 +90,45 @@ public static function upload_mime_types(array $mimes) {
   }
 
   /**
+   * Removes svg width and height attributes from the content.
+   *
+   * @implements the_content
+   */
+  public static function the_content($content) {
+    $content = preg_replace_callback('@<img [^>]*src=.+?.svg[^>]*>@', function ($matches) {
```

Comment:
Why is it needed to be replaced? I want to match everything in between the src attribute which ends on .svg. Any specific reason why this could not be achieved in this way? In addition with `.+?` the regex takes only 348 steps while the regex with `[^>]+?` takes 427. Not that significant. Just pointed out.